tests/test_order.py

```diff
@@ -27,16 +27,18 @@ def setup_class(cls):
         cls.item1 = Item.create(
             uuid=str(uuid.uuid4()),
             name='Item one',
-            price=10,
+            price=15,
             description='Item one description',
             category='Category one',
+            availability=23
         )
         cls.item2 = Item.create(
             uuid=str(uuid.uuid4()),
             name='Item two',
             price=10,
             description='Item two description',
             category='Category one',
+            availability=17
         )
 
         app.config['TESTING'] = True
```

Comment:
E non vedo assert nuovi, quindi in realta' non stai testando niente della tua feature.
Dovresti controllare l'availability corretta nei casi:
- creazione order
- modifica order
- cancellazione order